src/pudl/cli.py

```diff
@@ -124,20 +122,9 @@ def main():
 
     etl_settings = EtlSettings.from_yaml(args.settings_file)
 
-    if not (os.getenv("PUDL_OUTPUT") and os.getenv("PUDL_INPUT")):
-        logger.warning(
-            "PUDL will attempt to use legacy settings to derive paths."
-            "In the future this functionality will be deprecated in favor"
-            "of environment variables PUDL_OUTPUT and PUDL_INPUT. For more"
-            "info see: https://catalystcoop-pudl.readthedocs.io/en/dev/dev/dev_setup.html"
-        )
-        pudl_settings = pudl.workspace.setup.derive_paths(
-            pudl_in=etl_settings.pudl_in, pudl_out=etl_settings.pudl_out
-        )
-
-        os.environ["PUDL_INPUT"] = pudl_settings["data_dir"]
-        os.environ["PUDL_OUTPUT"] = pudl_settings["pudl_out"]
-        os.environ["DAGSTER_HOME"] = pudl_settings["pudl_in"]
+    # Make sure environment is properly configured
+    with (Path.home() / ".pudl.yml").open() as f:
```

Comment:
Should we automatically look for this yaml file in `get_settings` if it's `None`?

Reply:
Hmm, that might be cleaner. I'll implement.